apps/docs/content/components/LogoSuite/react.mdx

```diff
@@ -411,6 +329,88 @@ Logobars using the `emphasis` variant will appear with relatively higher contras
     </svg>
   </LogoSuite.Logobar>
 </LogoSuite>
+<br />
+<LogoSuite>
+  <LogoSuite.Heading visuallyHidden>Muted</LogoSuite.Heading>
```

Comment:
To avoid it looking like we're stacking logobars again, can we make this look more like the examples above and use the description to explain what's happening?

E.g.

```suggestion
  <LogoSuite.Heading visuallyHidden>Muted example</LogoSuite.Heading>
  <LogoSuite.Description>Muted:</LogoSuite.Description>
```

Reply:
Same would also need to happen for Emphasis example